examples/fluent-bit/log-driver-buffer-limit/README.md

```diff
@@ -0,0 +1,28 @@
+### FireLens Example: Specifying buffer limit size with 'Fluentd' log driver
+
+`log-driver-buffer-limit` option is now supported on ECS EC2 and ECS Fargate with PV1.4. This setting helps FireLens to configure the Fluentd Docker Log Driver field [fluentd-buffer-limit](https://docs.docker.com/config/containers/logging/fluentd/#fluentd-buffer-limit). It will specify the limit size for number of events buffered on the memory to avoid log loss issue when buffer is full. 
+
+By default, the buffer limit will be set to `1MiB`. In order to increase or decrease the size, you can customize the `log-driver-buffer-limit` option in your log configuration:
+
+```
+	"logConfiguration": {
+		"logDriver":"awsfirelens",
+		"options": {
+			"Name": "cloudwatch",
+			"region": "us-west-2",
+			"log_key": "log",
+			"log_group_name": "/aws/ecs/containerinsights/$(ecs_cluster)/application",
+			"auto_create_group": "true",
+			"log_stream_name": "$(ecs_task_id)",
+			"log-driver-buffer-limit": "2097152"
+		}
+	},
+```
+
+The unit for this field is `byte` so above task definition will set the buffer limit size to `2MiB`.
+
+*Note*: 
+- PV1.3 is on deprecation. The feature is only supported after PV1.4.
+- The value for `log-driver-buffer-limit` should be an integer between 0 and 536870912 (`512MiB`).
+- The total amount of memory allocated at the task level must be greater than the amount of memory allocated for all containers in addition to the memory buffer limit for the FireLens log driver.
+- The total amount of memory buffer specified must be lesser than 536870912 (`512MiB`) when the container `memory` and `memoryReservertion` values aren't specified.
```

Comment:
nit: this is kind of confusing wording. It must always be less than 512 MB right? Regardless of the other settings, that's always the hard limit

Reply:
yeah so the last line is for if we have multiple containers have this settings, then their sum should be less then 512MB. like one container with `log-driver-buffer-limit`: `300MB` is allowed. But it is not allowed if we have two with 300MB (300 *2 > 512). But I do agree this sentence is kind of confused. Let me reword it

Reply:
Ah I see, I didn't read it correctly